README.md

```diff
@@ -1,2 +1,60 @@
+  - [Stacks Used](#stacks-used)
+  - [Contributors](#contributors)
+  - [Progress](#progress)
+  - [Roadmap](#roadmap)
+
+## Game Design
+
+Although this section can be read in any order, the explanation for how this game is conceived is best read from top to bottom, starting with the [Philosophy and Influences](#influences).
+
+### Philosophy and Influences
+
+"Grinding" is a controversial mechanic in game design. Detractors argue that by arbitrarily increasing the amount of time it takes for players to reach a goal disrespects the player's time, and waters down the gaming experience into an unmeaningful repetition of actions. Conversely, some players enjoy the long and steady work towards a goal, and sense of *pride and accomplishment* outweighs the possible troughs in the gaming experience.
+
+The popular **Monster Hunter** video game series features heroic players who wield over-the-top, menacing weapons to face-off against even more menacing enemies, which typically include massive, dinosaur-esque creatures.
+
+Here, grinding is a core mechanic where players repeatedly slay and capture monsters to obtain their materials to craft new equipment. Said equipment feature special properties, such as increased fire protection, or increased damage against certain types of enemies. This creates a feedback loop where players slay monsters, craft new gear, and try to slay monsters even more efficiently with their gained experience and new items.
+
+This Discord game aims to streamline the intensity of this experience into a more strategic experience, where players similarly fight off against powerful enemies, but in turn-based combat. By nature, the game will have long "reach" goals for the player to work towards, while being a more casual experience.
+
+Thanks to the power of Discord, many digital communities are made between friends and similarly-minded gamers. This allows this game to be very accessible to individuals, groups of friends, or "hardcore" players who might want to team up together to tackle the hardest challenges the game has to offer.
+
+### Game Loop
+
+Players control an adventuring company of customizable characters. They venture off alone or with a party of other players to fight monsters.
+
+Combat is turn-based, with various mechanics including cooldowns and skill variety (think **Pokémon!**) on both the players' and enemy's part, creating a situation where fights should be pre-planned and tactical decisions being made during a fight.
+
+Once a monster is slain, the participating players gain monster-specific crafting material in which they use to craft new gear, be it armours or weapons.
+
+With new gear worn and in hand, players venture off to take on another battle!
+
+## Stacks Used
+
+This game is being developed using **Node.js** and **PostgreSQL**. The game is played over **Discord**, and it takes advantage of the npm package `discord.js` to allow the game's bot to describe the game world and interact with players.
+
```

Comment:
Maybe add a section here on "how to set yourself up for development", linking to things you need to download and describing what files to create and how to initialize database and stuff like that

Reply:
Sounds like a next iteration thing, but aye that's a good call.